pkg/server/server.go

```diff
@@ -119,6 +121,13 @@ func (c *ProxyClientConnection) send(pkt *client.Packet) error {
 			return err
 		} else if pkt.Type == client.PacketType_DIAL_RSP {
 			if pkt.GetDialResponse().Error != "" {
+				body := bytes.NewBufferString(pkt.GetDialResponse().Error)
+				t := http.Response{
+					StatusCode: 400,
```

Comment:
This doesn't seem like a 400/Bad Request. 
Sadly we don't have the error passed back through the packet at this point.
I would suggest we should probably just use [500/Internal Server Error](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/500) or [503/Service Unavailable](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/503)